src/BackgroundTask/MonitorCssTransientCaching.php

```diff
@@ -239,38 +231,12 @@ public function query_css_transient_count() {
 	public function handle_plugin_update( $old_version ) {
 		// Note: We cannot use the is_css_transient_caching_disabled method because we need to get the underlying stored value.
 		$disabled = AMP_Options_Manager::get_option( Option::DISABLE_CSS_TRANSIENT_CACHING, false );
-		if ( empty( $disabled ) ) {
+
+		if ( $disabled ) {
```

Comment:
Why isn't this `empty( $disabled )` anymore? Wouldn't this invert the condition?

Reply:
We have removed the `handle_plugin_update()` now.